src/Factory/GiiFactory.php

```diff
@@ -20,7 +20,7 @@ public function __construct(array $generators = [])
     public function __invoke(ContainerInterface $container): GiiInterface
     {
         $generators = array_merge(
-            $container->get(Parameters::class)->get('gii.generators'),
+            $container->get(Parameters::class)->get('gii.generators', []),
```

Comment:
I'd accept generators array as constructor argument instead. See https://github.com/yiisoft/yii-demo/pull/91

Reply:
Yes, will do in the next PR.